src/coreclr/pal/src/include/pal/palinternal.h

```diff
@@ -163,9 +163,6 @@ function_name() to call the system's implementation
 
 /* C runtime functions needed to be renamed to avoid duplicate definition
    of those functions when including standard C header files */
-#if !defined(_DEBUG)
```

Comment:
The `#define memcpy DUMMY_memcpy` shoudl stay (without the #if). It represents the case when a function is not implemented in PAL and just forwarded to the platform implementation. We need the dummy definition in this case. See 
https://github.com/dotnet/runtime/blob/9e1350eb6d8d6018d599b7fefbae6805dbd420ea/src/coreclr/pal/src/include/pal/palinternal.h#L25-L51

Reply:
I believe this is causing the MUSL build legs failures